public/app/plugins/datasource/loki/components/LokiExploreExtraField.tsx

```diff
@@ -17,12 +17,12 @@ export function LokiExploreExtraField(props: LokiExploreExtraFieldProps) {
   const { label, onChangeFunc, onKeyDownFunc, value, type, min } = props;
 
   return (
-    <div className="gf-form-inline explore-input--ml">
```

Comment:
This class only added a 4px right-margin to LokiExploreExtraField component. I have removed it and instead of that added  4px left-margin to the div wrapping Query field. This keeps all fields aligned at the beginning of the row.